app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
+  has_and_belongs_to_many :projects
+  has_many :contributions
```

Comment:
A user has many contributors? 🤔

Reply:
@cone Those are _contributions_ (related to a project)